rebuild/src/public_key.cc

```diff
@@ -1,4 +1,4 @@
-#include "public_key.h"
+#include "addon.h"
```

Comment:
Couldn't resolve circular dependency without building/linking headers and implementations separately.  All `.cc` files import `addon.h` and `addon.h` was built with other headers in correct order to get everything to compile.